bit/network/services.py

```diff
@@ -268,6 +320,9 @@ class NetworkAPI:
     GET_TRANSACTIONS_MAIN = [BitpayAPI.get_transactions,  # Limit 1000
                              SmartbitAPI.get_transactions,  # Limit 1000
                              BlockchainAPI.get_transactions]  # No limit, requires multiple requests
+    GET_TRANSACTION_BY_ID_MAIN = [BitpayAPI.get_transaction_by_id, # Limit 1000
+                                    SmartbitAPI.get_transaction_by_id, # Limit 1000
+                                    BlockchainAPI.get_transaction_by_id]  # No limit, requires multiple requests
```

Comment:
Do we need the comments since we're just getting the tx? Also, let's fix indent 🙂

Reply:
Changes made.